### PR TITLE
update view template for some github events

### PR DESCRIPTION
**Pull request by pxthinh**

### CODE REVIEW

File: resources/views/events/github/issue_comment/created.blade.php [hunk 1]
```diff
@@ -7,13 +7,17 @@
...
+        'user' => "<a href='{$issue->user->html_url}'>@{$issue->user->login}</a>"
+    ]
+) !!}
 
 📢 <b>{{ $issue->title }}</b>
 
 @include('tg-notifier::events.shared.partials.github._assignees', compact('payload', 'event'))
 @include('tg-notifier::events.shared.partials.github._body', compact('payload', 'event'))
+@else
+IGNORE_MESSAGE
```
Comment (tanhongit) [inline resources/views/events/github/issue_comment/created.blade.php:22]: What is the effect of this "IGNORE_MESSAGE" string? Where it is handled. Please update and create a link if it needs to be processed in Core.

Also, I want it to be a constant. Or more versatile can be placed into an environment that is configured in the config, please.

File: lang/en/events/github/label.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?php
```
Comment (tanhongit) [inline lang/en/events/github/label.php:1]: This lang file has not been added for other languages

File: resources/views/events/github/issue_comment/created.blade.php (hunk 1 above)
Comment (pxthinh) [reply resources/views/events/github/issue_comment/created.blade.php]: This is the problem I encountered: https://github.com/cslant/telegram-git-notifier/issues/34

Comment (pxthinh) [reply resources/views/events/github/issue_comment/created.blade.php]: This is the problem I encountered: https://github.com/cslant/telegram-git-notifier/issues/34

**Pull request merged**